lib/auth/discoveryconfig/discoveryconfigv1/service.go

```diff
@@ -167,6 +177,21 @@ func (s *Service) CreateDiscoveryConfig(ctx context.Context, req *discoveryconfi
 		return nil, trace.Wrap(err)
 	}
 
+	if err := s.emitter.EmitAuditEvent(ctx, &apievents.DiscoveryConfigCreate{
+		Metadata: apievents.Metadata{
+			Type: events.DiscoveryConfigCreatedEvent,
+			Code: events.DiscoveryConfigCreateCode,
```

Comment:
```suggestion
			Type: events.DiscoveryConfigCreateEvent,
			Code: events.DiscoveryConfigCreateCode,
```

Use create or created consistently.

Reply:
Done